frontend/src/pages/MetadataByType/SingleMetaDataResult.jsx

```diff
@@ -0,0 +1,216 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Bubbles } from 'styled-icons/icomoon/Bubbles';
+import { StarFull } from 'styled-icons/icomoon/StarFull';
+import { StarEmpty } from 'styled-icons/icomoon/StarEmpty';
+import { MetadataURL } from '../../sharedComponents/Metadata/MetadataURL';
+import { ReleaseStateLabel } from '../../sharedComponents/Metadata/ReleaseStateLabel';
+import { alertActions } from '../../state/actions/alert';
+import { FeedbackLabel } from '../../sharedComponents/Metadata/FeedbackLabel';
+
+const SingleMetaDataResultContainer = styled.div`
+  margin: 0.5rem;
+  border: solid 0.2em #F7F9FA;
+  border-radius: 0.5rem;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  background-color: #f3f3f3;
+  max-width: 100%;
+  box-shadow: 0 0.1rem 0.25rem #dbdbdb; 
+`;
+const MetaDataContent = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.3rem;
+  background-color: white;
+  overflow: hidden;
+  max-width: 100%;
+`;
+const MetaDataDescription = styled.div`
+  flex-grow: 1;
+  padding: 0.5rem;
+  color: #313131;
+  overflow: hidden;
+  max-width: 100%;
+  & > p {
+    word-wrap: break-word;
+    position: relative;
+  }
+`;
+
+const MetaDataMunicipalityLink = styled(Link)`
+  & > p {
+    color: #3e3e3e;
+    font-size: 0.9rem;
+    background-color: #f2f2f2;
+    border-radius: 0.2rem;
+    padding: 0.2rem;
+    display: inline-block;
+  }
+`;
+const MetaDataLink = styled(Link)`
+  padding: 0.4rem;
+  display: inline-block;
+  background-color: #efd8ff;
+  border-radius: 0.2rem;
+  margin: 0.2rem 0;
+  color: rebeccapurple;
+`;
+const MetaDataRating = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 0.4rem;
+  background-color: #F7F9FA;
+`;
+const CommentsLink = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+
+  & > p {
+    font-size: 0.8rem;
+    max-width: 6rem;
+    text-align: center;
+  }
+  & > img {
+    max-width: 1.4rem;
+  }
+`;
+const Favourite = styled.button`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  border: none;
+  background-color: transparent;
+  padding: 0;
+  cursor: pointer;
+  & > p {
+    font-size: 0.8rem;
+    max-width: 6rem;
+    text-align: center;
+  }
+`;
+const CommentsIcon = styled(Bubbles)`
+  width: 1.4rem;
+  color: #5d5d5d;
+`;
+const StarFullStyled = styled(StarFull)`
+  width: 1.4rem;
+  color: #f9d205;
+`;
+const StarEmptyStyled = styled(StarEmpty)`
+  width: 1.4rem;
+  color: #5d5d5d;
+`;
+
+const SingleMetaDataResult = ({ metadata }) => {
+  const {
+    uuid, formatName, url, description, releaseState, experiencePostGuid, municipalityName,
+  } = metadata;
+
+  // TODO: Update this to use API whenever that exists.
```

Comment:
?

Reply:
I tried fixing this now, but realized it's not even part of this user story. It's also going to take more time than we have right now, so it'll have to wait until next sprint.

Reply:
I'm just confused as to what this TODO means to begin with

Reply:
TODO is just a code convention that most IDEs pick up on.

Reply:
<img width="723" alt="image" src="https://user-images.githubusercontent.com/23093283/76128263-1a8ff880-6004-11ea-9a6b-015435c9d289.png">
Example

Reply:
I am familiar with _what TODO means_ I was just confused as to what _this TODO means_ - i.e. that it refers to the amount of comments.

Reply:
Either way, this is fine, move on to my other comments